docs/src/windows.md

```diff
@@ -0,0 +1,106 @@
+# Install and contribute to Erdapfel on Windows
+
+## Windows 7 / 8
+
+- Install React Dev Tools in your browser ([chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) / [Firefox](https://addons.mozilla.org/fr/firefox/addon/react-devtools/))
+- Install [VirtualBox](https://www.virtualbox.org/)
+- Mount [this xubuntu image](http://cdimages.ubuntu.com/xubuntu/releases/18.04/release/xubuntu-18.04.1-desktop-amd64.iso)
+- Allocate at least 2 CPU, 4GB of RAM and 20GB of HDD and material acceleration to the VM
+- In settings > Network, choose Bridge Adapter
+- Inside the VM, install an IDE (enable ESlint plugin), open a terminal
```

Comment:
Same remark for the IDE, this is off-topic. If someone wants to contribute with Notepad and no realtime ESLint hint, technically they can :)